apps/frontend/src/libs/components/select/select.tsx

```diff
@@ -48,6 +50,39 @@ const Select = <TFieldValues extends FieldValues, TOptionValue>({
 		isLabelHidden && "visually-hidden",
 	);
 
+	const handleChange = useCallback(
+		(
+			selectedOptions:
```

Comment:
How about making it easier and working only with values? And I did it without `onChange`, it can still work correctly
```
	const handleChange = useCallback(
		(
			selectedOptions:
				| MultiValue<SelectOption<TOptionValue> | undefined>
				| SingleValue<SelectOption<TOptionValue> | undefined>,
		): void => {
			const selectedValues = isMulti
				? (selectedOptions as MultiValue<SelectOption<TOptionValue>>).map(
						({ value }) => value,
					)
				: (selectedOptions as SingleValue<SelectOption<TOptionValue>>)?.value;

			field.onChange(selectedValues);
		},
		[field, isMulti],
	);

```

but for this we need to do changes in table pagination:
```
type FormData = {
	pageSize: number;
};
````

Reply:
It was really overcomplicated, but this change in table pagination solves the issue. Thanks!